servant-client/test/Servant/ClientTestUtils.hs

```diff
@@ -74,6 +82,22 @@ instance FromForm Person
 instance Arbitrary Person where
   arbitrary = Person <$> arbitrary <*> arbitrary
 
+instance MimeRender PlainText Person where
+  mimeRender _ = LazyByteString.fromStrict . encodeUtf8 . Text.pack . show
+
+instance MimeRender ctype a => MimeRender ctype (WithStatus _status a) where
```

Comment:
Shouldn't these instances be in the library itself?

Reply:
a8734ea20b516d5fd5f37a95ffec58a12bfbd36d

Reply:
sorry, wrong commit hash.  it's this one: a8734ea20b516d5fd5f37a95ffec58a12bfbd36d

thanks for the review!  :)